src/components/NcRichText/autolink.js

```diff
@@ -133,29 +136,21 @@ export const getRoute = (router, url) => {
 	const isAbsoluteURL = /^https?:\/\//.test(url)
 
 	// URL is not a link to this Nextcloud server instance => not an app route
-	if ((isAbsoluteURL && !url.startsWith(getBaseUrl())) || (!isAbsoluteURL && !url.startsWith(getRootUrl()))) {
+	if (isAbsoluteURL && !url.startsWith(getBaseUrl())) {
 		return null
 	}
 
-	// Vue 3: router.options.history.base
```

Comment:
Don't loose this commit

Reply:
It' already backported, so the comment didn't look useful anymore for me